cli/serve/serve.go

```diff
@@ -192,8 +194,31 @@ func serverMain(args []string, c cli.Config) error {
 	registerHandlers()
 
 	addr := net.JoinHostPort(conf.Address, strconv.Itoa(conf.Port))
-	log.Info("Now listening on ", addr)
-	return http.ListenAndServe(addr, nil)
+
+	tlsCA := conf.TLSCertFile
+	tlsKey := conf.TLSKeyFile
+
+	if tlsCA == "" || tlsKey == "" {
+		log.Info("Now listening on ", addr)
+		return http.ListenAndServe(addr, nil)
+	}
+
+	//config struct
```

Comment:
This comment is extraneous, but in general, comments should be complete sentences.